### DISCUSSION TITLE
Auf einmal keine Vebindung mehr zum HMS-1600

### DISCUSSION THREAD

**Discussion by Rustimation** [tbnobody/OpenDTU#1167] [Q&A] [answered]
Hi,
folgendes Phänomen: 
WR produziert nachgewiesenermaßen Strom, openDTU mit CMT Sendermodul zeigt seit Sonnenaufgang keine Verbindung mehr an, bzw. liest nichts mehr aus. Das openDTU Kästchen hat die letzten Wochen einwandfrei funktioniert. Sendeleistung auf max. hochgestellt, keine Änderung.
Kann es sein, dass der WR nach dem Aufwachen seine Funkverbindung verbaselt und dann nicht mehr connecten kann? Oder kennt jemand einen spontanen Ausfall der DTU?
Danke für jeden Hinweis
Chris

Nicht sehr erhellendes Log nachfolgend:
Websocket: [/livedata][6] disconnect
10:40:10.483 > RX Period End
10:40:10.483 > All missing
10:40:10.483 > Nothing received, resend whole request
10:40:10.483 > TX ActivePowerControl 868.00 MHz --> 51 84 65 64 15 80 15 66 04 81 0B 00 07 EE 00 00 21 60 14 
10:40:12.528 > RX Period End
usw...

bzw nach Neustart der DTU
Connected to MQTT.
10:48:41.484 > Fetch inverter: 116484656415
10:48:41.484 > Request SystemConfigPara
10:48:41.864 > TX RealTimeRunData 868.00 MHz --> 15 84 65 64 15 80 15 66 04 80 0B 00 64 BE 3A E9 00 00 00 00 00 00 00 00 E4 31 25 
10:48:42.087 > RX Period End
10:48:42.087 > All missing
10:48:42.087 > Nothing received, resend whole request
10:48:42.087 > TX RealTimeRunData 868.00 MHz --> 15 84 65 64 15 80 15 66 04 80 0B 00 64 BE 3A E9 00 00 00 00 00 00 00 00 E4 31 25 
10:48:42.899 > RX Period End
10:48:42.899 > All missing

**Comment by ms49434** [comment, marked as answer]
Wird in OpenDTU der CMT2300A als konfiguriert und verbunden angezeigt?
Schon mal die Frequenz vom CMT2300A auf 865 MHz gesetzt?
Greifen noch andere Geräte auf den Wechselrichter zu, z.B. eine Hoymiles Pro S DTU?
Was zeigt der LED Blink-Code vom Wechselrichter an?
Schon versucht die OpenDTU näher an den Wechselrichter zu rücken?
Antenne locker?
Wechselrichter S/N richtig?
Pin-Belegung in OpenDTU (GPIO) überprüft?

**Reply by Rustimation** [reply]
Danke für die Checkpoints. Das mit der Frequenz probiere ich beim nächsten Mal aus.
CMT2300A konfiguriert und verbunden laut Info Tab
andere Geräte greifen nicht auf den WR zu
Blink Code habe ich nicht, da ein anderes Modell ohne LEDs, evtl ein Grund zum Nachrüsten
Näher an die DTU heran geht nicht. Bisher hat es immer mit 17db Sendeleistung stabil funktioniert.
S/N ist unverändert, Pinbelegung ebenso.

**Momentan funktioniert alles wieder.** 
Der WR hat folgende Einträge im Log.

Begin | Ende | Fehlernummer | Meldung
-- | -- | -- | --
05:36:09 | 05:36:09 | 1 | Inverter start
05:36:16 | 00:00:00 | 212 | PV-4: No input
05:41:16 | 00:00:00 | 222 | PV-4: Input undervoltage
10:10:34 | 10:10:34 | 13 | Unknown
10:10:35 | 10:11:41 | 148 | Grid: Grid disconnection
10:12:17 | 10:12:17 | 2 | DTU command failed
10:12:56 | 10:12:56 | 13 | Unknown
10:12:56 | 10:14:29 | 148 | Grid: Grid disconnection
10:48:41 | 10:48:41 | 2 | DTU command failed
10:48:46 | 10:48:46 | 2 | DTU command failed

und dann noch mehrere  DTU command failed Einträge. Evtl. liegt's doch an der Funkstrecke... :-(

Ich frage mich nur, ob das öfter vorkommt, dass sich die Verbindung WG-DTU nach dem Neustart (Sonnenaufgang) nicht wieder herstellt.

**Comment by user231126** [comment]
Ein identisches Problem mit einem HMS-2000 habe ich ebenfalls. Nach Stromausfall war auf der openDTU jegliche Konfigurationseinstellung weg. 
Nach Neueinrichtung keine Verbindung mit dem Wechselrichter mehr möglich
2. openDTU besorgt und in Betrieb genommen - ebenfalls keine Verbindung
Anschließend DTU-WLite-S gekauft - innerhalb von 10min war die Verbindung da.
Hat man die Möglichkeit eine Art Sniffer Mode zu aktivieren ?
Parallelbetrieb geht ja nicht. Das konnte ich feststellen, dann funktioniert es auch mit der orginalen DTU nicht. Allerdings erst am nächsten Tag. Vermutlich weil Nachts keine PV Leistung vorhanden ist und am nächsten Morgen der Verbindungsprozess neu starten wird.
Die Profilliste bin ich durchgegangen. Bei mir blinkt am Wechselrichter die grüne LED. Ist normal, da nicht alle Eingänge angeschlossen sind.

**Comment by ollimucmac** [comment]
Bei mir tritt nach ca 3 Monaten Betrieb eines HMS 1600 4T mit OpdenDTU dasselbe Problem auf. Plötzlich keine Daten mehr vom WR. M.E. liegt es bei mir am Hoymiles. Wenn ich alle 4 Solarpanels abstöpsle und den AC-Stecker und 1 min warte, bis nichts mehr blinkt, dann wieder alles anstöpsle geht es wieder. Ohne die OpenDTU neu zu starten. Also klar ein WR Problem. Hat jemand dieselbe Symptomatik bzw. einen Fix?

**Comment by Gururom** [comment]
Manches mal, also so alle Monat mal, geht auch bei mir (HMS16oo 4T) die Verbindung verloren, nutze onBattery.
Denke auch es liegt am Wechelrichter, habe mit der DTU alles versucht, dichter ran, Konfig überarbeitet und und und 
Irgendwann gehts plötzlich wieder mal nach einer Stunde, mal brauchts einen ganzen Tag

**Comment by Rustimation** [comment]
Manche Threads leben länger :-)
Seitdem ich die Frequenz der openDTU auf 863MHz umgestellt habe, gibt es keine Ausfälle mehr. Weiß der Geier weshalb es damit funktioniert. Evtl. funken andere Geräte in der Nachbarschaft dazwischen...
Meine Fritz.Box macht manchmal Probleme: Mit dem iPhone oder iPad kann  ich gelegentlich nicht auf die OpenDTU zugreifen, auch nicht auf einen Raspberry Pi - mit dem PC klappt es aber. Könnte aber auch an einem WiFi Access Point liegen, der dazwischen hängt.

**Reply by ollimucmac** [reply]
Danke Rustimation, das ist ein toller Tipp, ich probier das, sobald der nächste Ausfall kommt. Zugriffsprobleme auf die OpenDTU kenne ich auch, bei mir liegt es evt. daran, dass die OpenDtu auf dem 2,4Ghz-Teil des WLANS läuft und von Geräten, die gerade mit 5 Ghz eingebucht sind, geht der Zugriff manchmal nicht, Oft aber doch. Stört mich nicht so sehr. Vllt. ähnlich wie mit Deinem Access-Point?

**Comment by Gururom** [comment]
Mit der Frequenz habe ich auch schon mal gehört, aber noch nicht versucht, im Logfile wird, glaub ich die Frequenz angezeigt auf der der WR funkt ??.
Verbindungsprobleme aufs Userinterface habe ich auch oft, liegt wohl wirklich an meinem Mesh und den diversen APs.
Am Pc der per Kabel an der Fritte ist klappts immer, mit dem Handy nicht.
Aber damit kann ich leben, mit den Abbrücjhen zum WR aber eben nicht

**Comment by ollimucmac** [comment]
Hi Zusammen, ich habe eine neue Idee, warum evt. der WR nach einer Woche die Verbindung zur OpenDtu verlieren könnte: Time Calibration. Habe vor ein paar Tagen auf die neueste Version upgedatet (hatte vorher eine von Herbst 2023) und nun taucht täglich im Evnt-Log ein Eintrag "Time Calibration" auf. Den gab es vorher nicht, vielleicht hat die alte OpenDtu-Version den WR nicht mit aktueller Zeit versorgt und bei zu großer Abweichung verstehen sich die Geräte nicht mehr? Seit dem Softwareupdate hatte ich keinen Verbindungsabbruch zwischen OpenDtu und WR mehr, das ist jetzt eine gute Woche her.

**Reply by Gururom** [reply]
Gibt soviele Gründe, aber keiner ist es wirklich im Endeffekt.
Habe trotz neuster Version einen Abbruch gehabt, 
wenn ich die Sendefrequence ändere auf 863, habe ich sofort Abbrüche.
wenn ich die Sendestärke ändere, ob nun 8 oder 20db passiert überhaupt nichts.
keine Ahnung, und wahrscheinlich viele andere auch nicht.

**Reply by Marc9485** [reply]
Kondensator als Puffer könnt ihr noch ausprobieren, grösseres Netzteil, USB-Kabel mit höherem Querschnitt

**Reply by Rustimation** [reply]
@Gururom: _"wenn ich die Sendefrequence ändere auf 863, habe ich sofort Abbrüche._"
Du must dem HMS-1600 schon etwas Zeit geben, die neue Frequenz zu finden. Das kann u.U. 20 Minuten dauern. Probiere es mal mit mehreren anderen Frequenzen - nicht alle sind erlaubt, aber ich glaube die openDTu weist dich darauf hin, welche gehen und welche nicht.

**Reply by Gururom** [reply]
Netzteil / Kabel habe ich vor langer Zeit bereits getestet, wie gesagt, die Abbrüche liegen zu 99%iger Sicherheit am WR.
Frequence kann schon sein wie du sagst, aber wenn ich über 4 Wochen null Probleme habe, meine oDTUoB alle 5 Sec syncronisiert, keinerlei Auffälligkeiten in der Logdatei, glaub ichs einfach nicht.
Da kommt spontan und aus dem Nichts ein Abbruch, welcher sich für eine undefinierte Zeit hält, und dann gehts plötzlich wieder wie als wenn es nichts wäre.
Ich glaub das nächste Mal mach ich einfach nichts und warte die Zeit ab....

**Comment by Paule-Panter** [comment]
Auch mein HMS1600-4t verliert sporadisch die Verbindung zur DTU bzw. andersrum.
Ich habe mal nachdem die Verbindung abgebrochen war, mit einem SpectrumAnalyser die Luft angeguckt.
In der DTU war zu dem Zeitpunkt die 865,75MHz eingestellt und hat auch schon seit Sonnenaufgang den halben Tag wunderbar funktioniert.
Nach dem die Verbindung verloren war, konnte ich aber diverse periodische Datenbursts im SpectrumAnalyser bei 865,25MHz sehen. Daher habe ich die DTU mal auf diese Frequenz umgestellt und abgespeichert und sofort war der Wechselrichter wieder online und lieferte problemlos wieder seine Daten.
Im Konsolenlog konnte man vorher sehen, dass die DTU immermal auf 868MHz schalten wollte, scheint irgendein Defaultwert zu sein.
Ich frage mich, ob der WR selbst auch nach ungestörten Kanälen sucht, und daher ggfs. auf einen anderen (ungestörteren) Kanal ausweicht? 
Eigentlich müsste die DTU nach Verbindungsverlust mal das Frequenzband durchsuchen um den WR wiederzufinden, oder sollte der WR die DTU finden? Wenn letzteres, dann scheint das nicht mehr richtig zu funktionieren. Im Spektrum sieht man die beiden jedenfalls aneinander vorbei reden.

**Reply by Rustimation** [reply]
Tolle Analyse. Da muss ich - mangels Equipment - echt passen.  IMHO scannt der HMSxxx das 868MHz Band wenn er sich nicht gleich verbinden kann  und sucht dann mit die DTU bzw. openDTU. 
D.h. verstellt man die Frequenz in der DTU, geht die Verbindung erst einmal verloren, kommt aber nach max. 20 Mínuten wieder. 
Es ist also genau so: Nicht die DTU sicht nach dem Wechselrichter, sondern der WR sucht die DTU. Der Trick dürfte sein, eine Frequenz in der openDTU zu finden, die wenig Störsignale hat. Auf die stellt sich dann der WR - zeitverzögert - ein. Ferner sollte man besser eine "legale" Frequenz auswählen.
VG Chris

**Reply by Gururom** [reply]
Also kann es sein, dass wenn man die Verbindung verliert irgend etwas die Frequence stört , aber der Verlust der Verbindung passiert ebnen auch bei Leuten die die DTU dicht neben dem WR haben, da sollten Störungen keine Rolle spielen.
Man sollte außerdem rausbekommen, ob das ein Problem bei allen ist, oder nur bei einer kleinen Minderheit.
Erschwerend kommt hinzu, dass viele Nutzer der openDTU solche Ausfälle warscheinlich gar nicht mitbekommen.
Nutzer von openDTUoBattery (wie ich zBsp.) eher doch, da die ja aktiv mit der Regelung arbeiten.
Es wäre also gut, wenn es eine Protokollierung in der DTU geben würde, die zu gegebenen Zeiträumen nachgeschaut werden könnte.

**Reply by Paule-Panter** [reply]
Das Equipment ist heutzutage nicht wirklich mehr teuer. ;-)
Ein dvb-t usb-stick der am handy oder tablet angesteckt als software defined radio (SDR) genutzt wird.
Dazu die passende Software (Stichwort SDR) und fertig ist die notwendige Ausstattung.  SDR-Stick mit rtl Chipsatz kostet bei Amazon rund 40€, bei aliexpress sicherlich für die Hälfte zu haben. SDR Software gibt es als Testversion kostenlos,  wenn man es etwas intensiver nutzt,  sind die 9€ sicherlich ihr Geld wert. 
![Screenshot_20240712_073518_SDRTouch](https://github.com/user-attachments/assets/31c6cfd1-4236-451d-b220-e41b49edb88f)

Hat auch den Vorteil,  dass man sieht welche frequenz derzeit kaum gestört ist,  bzw komplett gestört sind.  Bei mir sind einige Frequenzen dauerhaft mit irgendwelchen Signalen fett belegt.(nicht in diesem Screenshot)

**Reply by Martinovici** [reply]
Hallo, habe die selbe Methodik wie Paule Panther versucht. bei mir war es allerdings 868,25MHz.

**Hintergrund**: Ich habe das gleiche Problem, dass alle paar Tage am Morgen keine Verbindung mehr besteht. Ein DTU-Reset oder das Senden eines Resets an den Wechselrichter funktioniert nicht mehr. Die TX-Periode endet ohne Antwort vom Wechselrichter.
![image](https://github.com/user-attachments/assets/a3ebfade-3e50-414c-91aa-b605505d8f08)
Am Spektrum sind nur die "Beacons" des DTU zu sehen
![image](https://github.com/user-attachments/assets/22869688-a9f8-46b2-aa5d-bdb54fa6b20c)
Bisher hat nur funktioniert, wenn ich den WR komplett vom Strom nehme (inkl. Panels)


Nun nach der Suche nach einem Signal des WR bin ich auf 868,25 gestoßen. DTU darauf gestellt. erst nichts gesehen (nur ein paar Sekunden gewartet). Dann hab ich noch ein Reset mittels DTU an WR gesendent. (funktioniert normalerweise ja nicht) und siehe da:
![image](https://github.com/user-attachments/assets/72a097df-6091-4fc7-994c-e1c8c464ac8b)
der WR antwortet wieder :-)

Ich melde mich erneut sobald der WR wieder stumm wird.

**Comment by jaybanane** [comment]
Super spannend, dass Ihr da langsam was dazu rausfindet.
Ich habe dieses Problem auch schon lange. Alle paar Tage verliert der Inverter völlig random die Verbindung zur DTUoB. Manchmal war sie dann Tage lang nicht verbunden. Was super gefährlich ist, weil der zuletzt eingestellte Wert im Inverter dann bestehen bleibt. Das kann z.B. die Batterie killlen, zumindest bis am BMS der Niederspannungsschutz greift. Wäre aber super ungesund für die Batterie. Die einzige Möglichkeit die ich fand, dass die Verbindung wieder aufgebaut wird, ist die Trennung der DC Seite des Inverters für 30s+, bis die LED aus geht und dann wieder anschließen.
Dabei finde ich diese Lösung sehr unschön, da beim Anschließen des Inverters an die Batterie ja enorm hohe Ströme fließen, bis die Kondensatoren aufgeladen sind. Über kurz oder lang soll das den Inverter killen. Über einen Widerstand anzuschließen hatte ich auch schon gelesen, aber noch nicht umgesetzt, da auch schwer zu automatisieren. Zumindest habe ich aktuell folgende fall back Lösungen:
Mein Home assistant schaltet die AC Seite des Inverters über eine Wifi Steckdose aus, falls SOC von Victron smart shunt oder Spannung von Victron Laderegler zu niedrig sind. Zudem überwache ich den Status ob der Inverter erreichbar ist und falls er es für ein paar Minuten nicht ist, dann lasse ich via home assistant die DC Seite des Inverters über ein Tuya DC relais trennen und nach 3 Minuten  wieder verbinden. Das hilft und der Inverter wird immer direkt wieder erreichbar, aber somit hat man jedes mal die unschöne Trennung der DC Seite.
Was ich bisher versucht habe um den Fehler zu beheben (bisher alles ohne Erfolg):
Umstieg auf andere DTU Hardware, von Einzelkomponenten Eigenbau zu DTU fusion
Verschiedenste Einstellung der Sendeleistung, verschiedene Antennen (Abstand DTU zu Inverter ca 30cm)
Verschiedene Inverter (Fehler ist sowohl bei HMS800 als auch HMS1600 vorhanden)
Änderungen im WLAN Mesh Netzwerk
Verschiedenste DTU Firmware Versionen

Ich denke der Fehler liegt tatsächlich bei den Invertern oder den verwendeten Frequenzen, wie von Euch berichtet... Bin gespannt auf jegliche Updates von Euch.
VG Jürgen

**Reply by Gururom** [reply]
30cm zwischen DTU und WR ist natürlich sehr dicht.
Habe schon oft gelesen dass auch das Probleme macht.
Ich habe meine Fusion ca 15-20m und ca 4 Höhenmeter entfernt hinter einer ca 40cm dicken Wand entfernt vom WR zu stehen und in 99% aller Fälle keine Verbindungsprobleme, wird nach einstecken innerhalb von 1Minute gefunden.
Trotz alledem hatte ich bisher so ca alle Monat bzw 1,5 Monate mehr odfer weniger einen ganzen Tag mit Abbrüchen und Totalverlusten zu tun.
Meine Fusion ist neu und erst seit ein paar tage ohne Probleme in Betrieb.
Ich werde das weiter testen, habe aber auch den WR in Verdacht

**Comment by Fokker-1** [comment]
Hallo,

ich klicke mich hier mal als "Mitleidender" hinzu ...  :-(
Danke für diesen kompetenten Thread, wirklich super.

Zu meinem Nulleinspeise-System.
Ich habe seit 2 Wochen eine Nulleinspeisung mit Akku aktiv, bestehend aus folgenden Komponenten:

1) Lader:
Meanwell NPB-1200-24 mit Trucki-Stick, Akku LiFe 25,6 V / 100A.

2) Netzeinspeisung:
HMS-1000-2t und openDTU on Battery.

3) Zählerdaten (eHz) auslesen:
über Hichi IR-Leser (Tasmota)

Alle FW ist auf aktuellem Stand.

Die Ladefunktion funktioniert prima.
Das Einspeisen vom Akku eigentlich auch - aber auch ich habe diese sporadischen Abbrüche.
Einen Zusammenhang mit irgendwelchen Ereignissen konnte ich bisher nicht finden, habe schon viel Tests und Anpassungen gemacht.

Dazu mal kurz vorab noch eine Frage:
Mein HMS-1000 an meiner eigentlichen PV-Anlage hat u. a. auch folgende Blinkfrequenz: Abstand 4 sec in grün.
Gleiches auch der aktuelle HMS-1000 zur Nulleinspeisung.

Gemäß Manual gibt es die aber nicht.
Im Frühjahr hatte ich deswegen mal bei der Hoymiles Hotline nachgefraget.
Ergebnis: kennt man bei Hoymiles nicht ... (???)

Habt Ihr ein solches Verhalten auch?

Wenn diese Blinksequenz auftritt, habe ich zwar weiterhin Verbindung zum WR (die Anzeige auf dem Monitor ist gelb), aber der WR aktiviert keine Einspeisung mehr, das geht oft 15 Minuten so. Oft endet das auch mir einem Reset, was dann die Tagesdaten löscht.

Nun habe ich aus Euren Untersuchungen entnommen, dass eine Veränderung der Frequenz ggf. noch eine Lösungsmöglichkeit gegen die Abbrüche wäre. Hat das bei Euch dauerhaft funktioniert?

Ich habe das bei mir jetzt auch mal gemacht und von 865 MHz auf 863 MHz geändert, sonst fällt mir ohnehin nichts mehr ein und hoffe inständig, dass das eine Lösung ist. Ggf. werde ich dann noch weitere Wechsel machen - die Hoffnung bleibt ja.
Wäre ja nicht so schön, wenn man mit diesem Equipment doch reichlich Geld versenkt hätte.

Ihr hattet auch die Problematik angesprochen, dass der WR in der zuletzt befindlichen Position verbleibt, wenn die Verbindung weg ist. Das kann ich so bestätigen.
Auch ich sehe das als gefährlich an. Da kann man nur hoffen, dass das BMS im Akku auch wirklich greift. Aber klar, gesund ist das für den Akku sicher nicht.

Für die HMS-Typen gibt es bei den openDTU-Versionen (ich habe Fusion und on Battery) in der neueren FW die Funktion "Funkstatistik".
Das ist eigentlich super, aber leider kann ich mit den Begrifflichkeiten dort nicht viel anfangen.
Welche konkreten Schlussfolgerungen kann man aus Fehlermeldungen ziehen wie:
nichts empfangen / teilweise empfangen / beschädigt empfangen / Fragment-Wiederanforderung ...

Ganz wichtig wäre es m. E. auch, dass ein Datum mit ausgegeben würde, zu dem die erste Anfrage gesendet wurde, damit man bei der Analyse weiß, wann das System gestartet wurde.

Nun denn, ich hoffe, dass Eure tollen Hinweise mit der Frequenzänderung zu einer Problemlösung führen und melde mich wieder, wenn Ergebnisse vorliegen.

Noch mal danke und Grüße
Werner

**Comment by Paule-Panter** [comment]
Hallo, 

Meine DTU funktioniert mittlerweile eigentlich recht stabil.
Der WR ist auf dem Dach unter dem Panels montiert. Die DTU steht im Wohnzimmer,  zwei Etagen darunter.  Daher hatte ich auch die Sendeleistung erhöht. Die Frequenz habe ich auf 868,00MHz gestellt. Ich hatte das "Gefühl",  dass der WR auf dieser Frequenz zumindest das Frequenzwechselkommando erwartet. (Im Logfile der DTU war diese Frequenz zumindest immer angegeben,  wenn die DTU versuchte einen reconnect hinzubekommen). 
Da auf 868,00MHz bei mir in der Umgebung im Spektrum recht ungestört aussehen, habe ich die DTU einfach auf dieser Frequenz eingestellt. 
![Screenshot_20241026_160720_Chrome](https://github.com/user-attachments/assets/17f0e625-8abb-406b-86bc-605e3439b5e6)
Seitdem funktioniert das bei mir recht stabil. (Allerdings habe ich auch keine Nulleinspeisungssteuerung aktiv.) 
Auch im lokalen Logfile waren keine sachdienliche Hinweise enthalten.  Daher habe ich auch keine weiteren Nachforschungen mehr angestellt. 

Gruß Paule-Panter

**Comment by Fokker-1** [comment]
Hallo Paule-Panter,

danke für die Informatinen.

Ok., dann ist Deine Anwendung etwas anders als aktuell bei mir, ich habe ja die Probleme mit der openDTU on Battery und Nulleinspeisung.
Die DTU ist übrigens ein Fertiggerät aus dem DTU-Shop.

Allerdings habe ich an meinem PV-System ebenfalls noch einen HMS-1000-2t. Der läuft jetzt seit einigen Monaten problemlos auf 868 MHz.
Diese DTU dazu ist eine "Fusion", da damit auch noch ein HM-800 (2,4 GHz) parallel betrieben wird.

Ich hatte gestern ja bei der oDTU on Bat die Frequenz gewechselt, leider kein Erfolg. 
Nach ca. 15 Stunden zeigte die "Funkstatistik" folgende Ergebnisse:


![Funkstatistik](https://github.com/user-attachments/assets/bbdb0503-3705-4ecf-a3fe-305342d9829d)


Leider also kein Erfolg.

Besonders auffällig sind die hohen Werte bei "Fragment", das sind etwa 20 % der "Anfragen". Was das wirklich bedeutet, weiß ich nicht.

Bei "nichts empfangen" hatte ich gestern zufällig bei 3 dieser Ereignisse bemerkt, dass die in unmittelbarer, zeitlichen Nähe aufgetreten sind, als auch der WR ein "Reset / Neustart (?) ausgeführt hatte. Das ist irgendwie verdächtig.

Auch ist mir nicht so richtig klar, was genau z. B. "nichts empfangen" oder"beschädigt" oder auch "Gesendete Fragment" bedeuten, so dass ich daraus auch keine Indikation auf eine Fehlerursache ableiten könnte (... und was sind die genauen Unterschiede zwischen: teilweise / beschädigt / Fragment?).

Ich habe jetzt noch einmal die Frequenz umgestellt auf 868,75 MHz.
Wenn das auch nichts bringt, dann muss ich wohl aufgeben ...   :-(

Gruß und schönen Sonntag
Werner

**Reply by Rustimation** [reply]
Hi,
meiner Erfahrung nach bedeutet "Nichts Empfangen" auch, dass der Wechselrichter keinen Strom aus den Panels oder der Batterie bezieht. Die Hoymiles Teile laufen nicht auf Netzstrom sondern "leben" von dem, was auf der DC Seite reinkommt.
Bzgl. Frequenz: **Die** richtige Frequenz gibt es nicht. Je nachdem, welchen 868MHz Garagentorsender, welches 868MHz Babyfon, welche 868MHz Fernsteuerung für seine Drohne oder das Modellauto dein Nachbar hat, musst du per trial & error eine passende, störungsarme Frequenz suchen, die stabil funktioniert. Das ist zumindest meine Erfahrung/mein Halbwissen - ich bin kein Hochfrequenztechniker. Nach dem Einstellen einer neuen Frequenz dauert es bis zu 15 Minuten, bis sich der WR darauf eingestellt hat; d.h. er scannt das Frequenzband bis er einen Connect mit der DTU hat. 
VG
Chris

**Comment by Fokker-1** [comment]
Hallo,

leider tappe ich mit meinen Problemen immer noch ziemlich im Dunkeln:

Nulleinspeisung mit Akku (25 V), HMS-1000, openDTU on Battery, Hichi IR-Leser am Zähler (eHz).
Es gibt immer wieder Resets des Inverters.

Frequenzänderungen brachten keine Verbesserung.
Dann habe ich mal den Akku nur noch an einen String gemacht, dass wurde besser, hat die Resets aber nicht gelöst.

Die nächste Idee war dann, zwischen Akku und String-Eingang einen kleinen Widerstand zu schalten.
Warum: der Akku hat einen extrem geringen Innnenwiderstand, ein Solar-Panel aber nicht. Vielleicht hat der HMS ja damit ein Problem bei seiner MPPT-Regelung.

Möglich ist natürlich nur ein sehr kleiner R, sonst hat man keine Leistung mehr im String.
Ich habe mal ca. 62 mOhm verwendet, den habe ich mir zum Testen aus einem entsprechend langen Kabel 1,5 mm2 gemacht.

Das ging schon mal deutlich besser (immer noch nur ein String am Akku), kein Reset mehr in 36 Stunden, bisher waren es ohne R so um 10 Resets des HMS.

Warum der HMS aber immer mal die Einspeisung einstellte (kein Grund dafür vorhanden), weiß ich nicht.

Ungeachtet dieser Situation mal eine andere Frage.
Ich habe bei "Inverter Gesamtleistung" sehr oft kleinere Werte als beim Netzbezug, also z. B. Inverter-Leistung = 40 W, Netzbezuig = 75 W.

Hat jemand mal eine vollständige Konfiguration des "Dynamic Powr Limiter (DPL)" des HMS mal für mich?
Dann könnte ich mal vergleichen, ob ich da eine unglückliche Konfiguration habe.

Vielen Dank im Voraus und Grüße
Werner

**Comment by Rustimation** [comment]
Der Reset des WR hat eigentlich nichts mit der DTU zu tun. d.h. die Frequenzwahl wird daran nichts ändern. Oder meintest du mit "Reset" die verloren gegangene Verbindung zur DTU?
Nur so eine Idee: Könnte es sein, dass die 24V Eingangsspannung nicht ausreichen, den WR sauber am Leben zu erhalten? Ich habe einen HMS-1600 an 4 Panels und gerade eben kommen gerade einmal 46W (DC) zustande (Schattenlage, vormittags 9:00 Uhr) Das schwächste Panel liefert 4,8W bei 29V Spannung (0,17A). 
Die HMS haben eine Aufwachspannung von 22V, 24/25V liegen knapp darüber. Minimalinput bein HMS WR ist 16V. Der Output ist dann allerdings sehr bescheiden. 
Hast du mal probiert, zwei Batterien hintereinander (48V) zu verwenden? Das hat zudem noch den Vorteil, dass die Leitungsquerschnitte nicht so kritisch sind und dass der WR sich weniger mühen muss, um die Spannung auf 230V hochzupumpen. 
Der HMS ist ja ein klassischer Solarpanel Wechselrichter, ihn mit einer Batterie zu betreiben (mache ich auch, aber mit 48V) ist lt. Hoymiles eigentlich nicht zulässig, auch wenn es funktioniert.

**Comment by Fokker-1** [comment]
Hallo,

danke für die Info.

Mit "Reset" meinte ich den WR.
Leider sind auch weiterhin immer wieder Resets des WR aufgetreten.

Ich habe mich jetzt entschlossen, diese Geräte-Kombi für die Nulleinspeisung mit Akku nicht mehr weiter zu verwenden.
Ich werde den HMS-1000 nun als WR beim PV-System einsetzen und damit den HM-800 ersetzen.

In der normalen PV-Betriebsart habe ich ja schon einen weiteren HMS-1000 seit einigen Monaten in Betrieb, der läuft da bestens und ohne jedes Problem. Den HM-800 einschließlich DTU werde ich verkaufen.

Für die Nulleinspeisung mit Akku überlege ich mir eine andere Lösung.

Der HMS-1000 ist nicht für die Akku-Betriebsart spezifiziert. 
Es kann natürlich sein, dass das mit 48 V alles funktioniert. Aber ich habe nur die 24 V (8 LiFe-Zellen) und will mir nicht noch einen 48 V Akku anschaffen, die Kosten sind ja erheblich (für mich).

Noch mal danke an alle für den freundlichen Support.

Grüße
Werner
